server/routes/induction/common/reasonsNotToGetWorkController.ts

```diff
@@ -18,6 +18,12 @@ export default abstract class ReasonsNotToGetWorkController extends InductionCon
   ): Promise<void> => {
     const { prisonerSummary, inductionDto } = req.session
 
+    // Check if we are in the midst of changing the main induction question set (i.e. from long route to short route)
```

Comment:
Not "e.g." here as it's only relevant to short route. I might reword this to make it clearer.